### ISSUE TITLE
Include auto-categorizing and sorting after install || update gems in Gemfile

### ISSUE THREAD

**Issue by deHelden** [bundler/bundler#7466]
At my current project we have about 350 gems. It's not so much but it's how it is..
Basically when somebody add new gem it's added to bottom of Gemfile. Without categorization and alphabetic order it's painfully hard to read for new people. 

Could you please somehow integrate with https://github.com/nikolalsvk/ordinare.
It would be really awesome to have pre-installed ability not to think about all this stuff.

I would really appreciate any callback for `ordinare` after running `bundle install || update`

**Comment by colby-swandale** [comment]
This would be most suited to a Bundler plugin that can update the `Gemfile` after a bundle install/update

**Comment by colby-swandale** [comment]
Hi! Thanks for the suggestion on ways to improve Bundler. This would be most suited to a Bundler plugin that can update the Gemfile after a bundle install/update.

**Comment by deivid-rodriguez** [comment]
Yeah, I don't think this belongs in bundler either. You can also use `rubocop` which has the style rule you're requesting, and more: http://www.rubocop.org/en/stable/cops_bundler/
